tests/meson.build

```diff
@@ -0,0 +1,50 @@
+configure_file(input : 'unit-test.h.in',
+               output : 'unit-test.h',
+               configuration : cdata)
+
+incdir = include_directories('../src')
+
+bandwidth_server_one = executable(
+    'bandwidth-server-one', 'bandwidth-server-one.c',
+    link_with : libmodbus,
+    include_directories : incdir
```

Comment:
I would just add `dependencies: libmodbus_dep` which handles both the link_with and the include_directories.